script/test-gemini-cli-e2e.sh

```
@@ -119,25 +124,25 @@ else
 echo ""
 
-# Test 6: Reverse cross-model (antigravity → gemini-cli)
-echo "Test 6: Cross-model session (antigravity → gemini-cli)"
-log_info "Step 1: Start with antigravity-gemini-3-pro-low..."
-timeout 60 opencode run -m google/antigravity-gemini-3-pro-low \
+# Test 7: Reverse cross-model (antigravity → gemini-cli)
+echo "Test 7: Cross-model session (antigravity → gemini-cli)"
+log_info "Step 1: Start with antigravity-gemini-3.1-pro-low..."
+timeout 60 opencode run -m google/antigravity-gemini-3.1-pro-low \
   "Say: ANTIGRAVITY_START" \
   2>&1 > /tmp/gemini-cli-e2e-reverse-s1.log || true
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)